src/less/form-item.less

```diff
@@ -111,6 +111,10 @@
     line-height: 1.1;
 }
 
+.form-item--required .form-item__label:before {
```

Comment:
koli spatnej kombatibilite by som to asi nestyloval, ale budeme to stylovat az v konkretnom projekte,
aby sa zrazu nezjavili '*' ked si niekto nainstaluje novu verziu a robije im to dizajn

- https://semver.org/ je to MINOR verzie a teda musi byt spatne kompatibilna